src/components/Instructions.component.js

```diff
@@ -5,16 +5,38 @@ import './../styles/instructions.css'
 
 class Instructions extends Component {
   render() {
-    return (
-      <div className="instructions">
-        Follow the instructions on the README to get started!
-        <img
-          className="instructions__logo"
-          src="https://uiuc.hack4impact.org/img/colored-logo.png"
-          alt="h4i logo"
-        />
-      </div>
-    )
+    const items = this.props.items
+    let newItems = []
+    for (let i = 0; i < items.length; i++) {
+      if (items[i].length <= 3) {
+        if (i % 2 == 0) {
+          let temp = items[i].toUpperCase()
+          newItems.push(temp + ' ')
+        } else {
+          newItems.push(items[i] + ' ')
+        }
```

Comment:
You should look into React's map function and replace this for loop with the filter and map functions. It would look something like this:
 .filter(item => item.length >= 3)
 .map((item, index) =>.....